src/tokens/ERC721/ERC721Base.sol

```diff
@@ -42,10 +42,14 @@ contract ERC721Base is
     PlatformFee,
     ReentrancyGuard
 {
+    //  @dev Owner of the contract (purpose: OpenSea compatibility, etc.)
+    address private _contractOwner;
```

Comment:
this will work! I would say that it may be neater to import Ownable.sol from solid state. and use the internal functions to set/transfer owner as it follows the diamond storage pattern but don't think it is crucial on the token contracts.

another approach is to return the first admin on the contract. Haven't tested this but it came to mind. Taking advantage of `members` being an [EnumerableSet](https://github.com/solidstate-network/solidstate-solidity/blob/a6b27feb7f4b3150038a67175f66994f7284e39a/contracts/data/EnumerableSet.sol#L37-L43)
```solidity
import {AccessControlStorage} from "@solidstate/contracts/access/access_control/AccessControlStorage.sol";

// ...in the contract

    function owner() external view virtual returns (address) {
        return AccessControlStorage.layout().roles[ADMIN_ROLE].members.at(0);
    }

```